roles/kubernetes-apps/ansible/templates/dashboard.yml.j2

```diff
@@ -166,6 +166,13 @@ spec:
           # If not specified, Dashboard will attempt to auto discover the API server and connect
           # to it. Uncomment only if the default does not work.
           # - --apiserver-host=http://my-address:port
+{% if dashboard_custom_flags is string %}
+          - {{ dashboard_custom_flags }}
```

Comment:
I would not do double 'format', it's an array and documented as an array.
it's easy enough to do ["my-argument"]

Remove this conditione please